pkg/interface/chat/src/js/components/settings.js

```diff
@@ -222,6 +222,7 @@ export class SettingsScreen extends Component {
               Convert this chat into a group with associated chat, or select a
               group to add this chat to.
```

Comment:
```suggestion
              Convert this chat into a group with associated chat.
```

should change the copy here